[docs] dgify External pipelines (#30642)
dagster-io/dagster: docs/docs/guides/build/external-pipelines/using-dagster-pipes/reference.md

> Note: We're using `os.environ` in this example, but Dagster's recommendation is to use <PyObject section="resources" module="dagster" object="EnvVar" /> in production.

Reviewer: ```suggestion
Note: We're using `os.environ` in this example, but we recommend using <PyObject section="resources" module="dagster" object="EnvVar" /> in production.
```
